src/System.Private.ServiceModel/tools/SelfHostedWcfService/Program.cs

```diff
@@ -274,6 +266,15 @@ static void Main()
             WebSocketHttpsDuplexTextBufferedTestServiceHost webSocketHttpsDuplexTextBufferedTestServiceHostServiceHost = new WebSocketHttpsDuplexTextBufferedTestServiceHost(typeof(WcfService.WSDuplexService), webSocketHttpsDuplexTextBufferedTestServiceHostbaseAddress);
             webSocketHttpsDuplexTextBufferedTestServiceHostServiceHost.Open();
```

Comment:
I agree we should find a better structure for these pairs in a new PR.  It would also offer the option to display them to the console. For future thought.